packages/rsocket-core/src/RSocketResumableTransport.js

```diff
@@ -156,8 +156,8 @@ export default class RSocketResumableTransport implements DuplexConnection {
     this._statusSubscribers = new Set();
   }
 
-  close(): void {
-    this._close();
+  close(error?: Error): void {
```

Comment:
I'm wondering now if we need to update the interface to also represent this option `Error`. The interface currently defines `close` as:

```
  /**
   * Close the underlying connection, emitting `onComplete` on the receive()
   * Publisher.
   */
  close(): void;
```

Reply:
I updated the doc comment in packages/rsocket-types/src/ReactiveSocketTypes.js
Or is there somewhere else I missed?

Reply:
Haha well, that's embarrassing. I'm 0 for 3. I'm reviewing the PR via the vscode GH plugin and when digging into the `DuplexConnection` source it showed me the head, not your changes, so looks like we are good to go there 👍 .

Also, for reference, you can click on the line numbers when reviewing the changes under the "files" tab, which will update the URL in the URL bar to be an absolute link to the line number you clicked on.

![image](https://user-images.githubusercontent.com/6305490/118378825-d26f4d00-b59b-11eb-9236-8dd316297006.png)

Reply:
No worries, no worries. :)

And thanks for the tip! I somehow didn't know that worked for PR diffs...